Tests/ML/test_download_upload.py

```diff
@@ -143,9 +143,16 @@ def _test_mount_for_lightning_container(test_output_dirs: OutputFolderForTests,
     download_path = test_output_dirs.root_dir / "downloaded"
     mount_path = test_output_dirs.root_dir / "mounted"
     if not is_lightning_model:
-        for path in [download_path, mount_path]:
-            path.mkdir(exist_ok=True)
+        for path in [download_path, mount_path, test_output_dirs.root_dir]:
+            # If destination folder exists, delete content to ensure consistency and avoid 'FileExistsError'
+            if (path / "train_and_test_data").is_dir():
+                shutil.rmtree(path / "train_and_test_data")
+
+            # Creates directory structure and copy data
+            shutil.copytree(full_ml_test_data_path("train_and_test_data"), path / "train_and_test_data")
```

Comment:
I have this pet peeve about repeated code, and in particular repeated constants. Would be good to store "train_and_test_data" in a variable.